.rubocop.yml

```diff
@@ -3,7 +3,8 @@ inherit_from:
   - .rubocop.rails.yml
 
 AllCops:
-  TargetRubyVersion: 3.1
+  SuggestExtensions: false
```

Comment:
Why false? Isn't it worth for us to know when an extension is available and manually disable it in the project-specific rubocop config if we want to?

Reply:
I'm fine doing it at the project specific level, but I find the recommendations to be noisy and not relevant outside of when we update rubocop

Reply:
that's a fair point. We really only want the recommendations *once* (which is why I mentioned manually disabling the extension).

I'm happy with this either way tbh